openbsd/OpenBSDMachine.c

```diff
@@ -129,7 +129,7 @@ void Machine_delete(Machine* super) {
 }
 
 static void OpenBSDMachine_scanMemoryInfo(OpenBSDMachine* this) {
-   Machine* host = &this->super;
+   Machine* super = &this->super;
```

Comment:
@natoscott Quick question re `host` vs `super`: Did we do a consistent rename of this previously and just miss that here? Using `super` is the "smaler" patch, but I have the feeling as if we were trying to go the direction of `host`. Fine either way, just consistent across platforms preferred.

Reply:
I suspect it was just missed here - in situations where it is the super, we've used super - there's plenty of cases where its just a pointer field in a struct, or local variable, though and in those cases we use host.

Reply:
@natoscott @BenBE should this be changed back to `host` then, I am guessing?

Reply:
Given that it's inside the `OpenBSDMachine_` namespace and we are pulling this from `&this->super` I'd tend to go with `super` for this variable. This note was mainly put as a question to @natoscott given that he did the PR that caused most of this renaming.